components/webdriver_server/lib.rs

```diff
@@ -2423,6 +2426,18 @@ impl Handler {
             Ok(())
         }
     }
+
+    fn focus_webview(&self, webview_id: WebViewId) -> Result<(), WebDriverError> {
+        let (sender, receiver) = ipc::channel().unwrap();
+        let msg = WebDriverCommandMsg::FocusWebView(webview_id, sender);
+        self.send_message_to_embedder(msg)?;
```

Comment:
Nit, no need for a temporary.
```suggestion
        self.send_message_to_embedder(WebDriverCommandMsg::FocusWebView(webview_id, sender)?;
```